docs/_templates/sidebar-link-items.html

```diff
@@ -0,0 +1,19 @@
+<nav class="bd-docs-nav bd-links">
```

Comment:
I think making this change in the napari-sphinx-theme is likely a better place for these changes.

Reply:
Either way would be fine - I understand @TimMonko' s argument about these links being specific to our pages but we have a bunch of napari-specific items in the theme as well 🤷🏻